build_cdt_recipes.py

```diff
@@ -290,8 +290,8 @@ def _main(
     dist_arch_slug,
 ):
     """
-    Build all CDT recipes for a given DIST_ARCH_SLUG (e.g. cos6-x86_64,
-    cos7-aarch64, etc.)
+    Build all CDT recipes for a given DIST_ARCH_SLUG (e.g. conda-x86_64
+    for post-CentOS CDTs, or cos6-x86_64, cos7-aarch64, etc. historically)
```

Comment:
Noticing that the new slugs don't have a version. Did we want to bake one in like we did with `cos6` & `cos7`? For example `conda-2_28-<arch>`

Reply:
Yes. See your comment on https://github.com/conda-forge/conda-forge.github.io/blob/5e57e30a11e91dc127aab1543b2cffe8ed8b3290/community/minutes/2024-05-15.md?plain=1#L74-L77